packages/app-page-builder/src/editor/plugins/elements/button/index.tsx

```diff
@@ -70,7 +70,8 @@ const buttonElementPluginsFactory = (args: PbEditorElementPluginArgs = {}) => {
                                 "center",
                                 DisplayMode.DESKTOP
                             ) as unknown as "flex-start" | "center" | "flex-end"
-                        }
+                        },
+                        action: { actionType: "link", href: "" }
```

Comment:
Why this change? Can you just add a comment?

Sorry for nitpkicking, just wanted to be 100% sure I understand what's going on here.